downstream/modules/platform/ref-hub-variables.adoc

```diff
@@ -201,6 +201,11 @@ Generating a new token will invalidate the existing token.
 Default = 63072000 seconds, or two years.
 | *`nginx_tls_protocols`* | Defines support for `ssl_protocols` in Nginx.
 
+Values available `TLSv1.2`, `TLSv1.3`
```

Comment:
More values are available but are considered weak and unsafe.
However, one could decide to enable them anyway if needed.

https://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_protocols